_data/navigation.yml

```diff
@@ -204,96 +206,24 @@ t:
         url: /cn/docs/quick-start-dataconsistency/
 
     users:
-      - title: 用户手册
+      - title: Java Chassis
         children:
           - title: Java Chassis系统架构
             url: /cn/users/
           - title: 环境配置
```

Comment:
You reserve the environment configuration, and is this part havs been included in the quickstart?  if not, how about move this part to  the quickstart.